storage-inventory/ops/README.md

```diff
@@ -2,118 +2,169 @@
 
 # CADC Storage Inventory Architecture and Deployment
 
-
-Read the entire document or jump to a relevant section below.
-
-
-### Table of Contents
-
-1. [Introduction](#introduction)
-    * [Key Storage Inventory Terms and Concepts](#key-storage-inventory-terms-and-concepts)
-    * [Storage Inventory Resources](#storage-inventory-resources)
-1. [Overview of a Storage Site](#overview-of-a-storage-site)
-    * [System Architecture](#system-architecture)
-    * [Standalone Storage Site](#standalone-storage-site)
-1. [Client Software](#client-software)
-1. [Deployment Prerequisites](#deployment-prerequisites)
-    * [Hardware Requirements](#hardware-requirements)
-    * [Software Requirements](#software-requirements)
-1. [Deployment](#deployment)
-1. [Healthchecks and Monitoring](#healthchecks-and-monitoring)    
-1. [FAQ](#faq)
-
-
 ## Introduction
+### Storage Inventory Terms and Concepts
+<a id='term-artifact'>**artifact**</a>  
+A representation of a file and its metadata in the SI database, specifically the _inventory.Artifact_ table (see the [SI data model](https://github.com/opencadc/storage-inventory/tree/master/storage-inventory-dm)).  The term _artifact_ is often used to refer to both the database representation of a file and the file itself as one thing, but when an SI application is acting on an _artifact_, it specifically refers to the database.
+- ➡️ an artifact _URI_ is a unique idenitfier for a file (or object) stored in SI.  The last path component of a _URI_ is always the _filename_.
+
+<a id='term-bucket'>**bucket**</a>  
+SI can organize units of work (number of files or artifacts) by `buckets` and applications can be configured to work on subsets of these buckets.  
+- Buckets are represented by hex strings, e.g. `a74`.
+- When applications refer to buckets on storage, it refers to the location (e.g., an S3 bucket or a directory) in which files are stored.  This is dependent upon your storage configuration.
+- When applications refer to buckets in the SI database, it refers to the _inventory.Artifact.uriBucket_ column in the database.  This column is populated with a random hex string on artifact creation.
+- ❗ _storage_ buckets and _uri_ buckets are not the same thing, e.g. files in _storage_ bucket `04c` are not necessarily the same as artifacts in _uri_ bucket `04c`.
+
+<a id='term-collection'>**collection**</a>  
```

Comment:
I could be persuaded about this.  We do use the term collection generically when talking to other groups, and I don't think it is a CADC-specific term.  It is, though, at a layer above SI since it refers to curated collections within the archive.

Reply:
Removed the definition, and tried to clarify the language later in the text where I used this term.